go/vt/vtgate/executor.go

```diff
@@ -680,6 +735,10 @@ func (e *Executor) handleShow(ctx context.Context, safeSession *SafeSession, sql
 
 	switch show.Type {
 	case sqlparser.KeywordString(sqlparser.COLLATION), sqlparser.KeywordString(sqlparser.VARIABLES):
+		if show.Scope == sqlparser.VitessMetadataStr {
+			return e.handleShowVitessMetadata(ctx, safeSession, destKeyspace, show.ShowTablesOpt)
```

Comment:
You're still passing in the keyspace here?

Reply:
Removed